android/app/src/main/res/layout/item_circuit.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <com.google.android.material.card.MaterialCardView
+        android:id="@+id/circuit_card_layout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="@dimen/xs_medium_element_padding"
+        android:layout_marginBottom="@dimen/xs_medium_element_padding"
+        android:clickable="true"
+        android:focusable="true"
+        app:cardCornerRadius="@dimen/xs_medium_element_padding"
+        app:cardElevation="@dimen/tiny_element_padding">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:padding="@dimen/xs_medium_element_padding">
+
+            <ImageView
+                android:id="@+id/circuit_icon"
+                android:layout_width="@dimen/circuit_icon_display"
+                android:layout_height="@dimen/circuit_icon_display"
+                android:contentDescription="@string/todo"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent"
+                tools:src="@drawable/ic_dumbbell" />
+
+            <ImageView
+                android:id="@+id/more_menu"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:contentDescription="@string/todo"
```

Comment:
Would filling in the descriptions help with accessibility?

Reply:
Gonna create a story for that, we should look into how content descriptions are properly written out before diving in. Also this is something Nimish can do for example, no need for devs to worry much about this kinda stuff.